external/test-runtime/optional.json

```diff
@@ -4,7 +4,9 @@
       "dependencies": {
         "Microsoft.DotNet.IBCMerge": "4.6.0-alpha-00001",
         "Microsoft.DotNet.UAP.TestTools": "1.0.2",
-        "TestILCNugetPackageForCoreFX": "1.0.0-beta-25304-01"
+        "TestILC.amd64ret": "1.0.0-beta-25305-00",
```

Comment:
Here it is: [EncryptedConfigNuGetRestore.cs](https://github.com/dotnet/buildtools/blob/426ce79360ff3530f9c6659425183939f2c4a23b/src/Microsoft.DotNet.Build.Tasks/EncryptedConfigNuGetRestore.cs). I don't know how NuGet package restore works under the covers in the msbuild world, but maybe EncryptedConfigNuGetRestore could be updated to support both.